src/modes/direct_select.js

```diff
@@ -67,9 +74,15 @@ module.exports = function(ctx, opts) {
         }
       });
       this.on('mouseup', () => true, function() {
+        if (dragging) {
+          ctx.map.fire(Constants.events.UPDATE, {
+            type: Constants.updateTypes.CHANGE_COORDINATES,
```

Comment:
`type` is used by `mapbox-gl-js` to store the event fired type so when this event gets to the application its type is `draw.update`. Maybe action or reason will work?